gc_little_helper_II.user.js

```diff
@@ -12460,6 +12465,9 @@ var mainGC = function() {
             cssCff('cdos', '14', '200', '460', '54');
             html += newParameterVersionSetzen("0.10") + newParameterOff;
             html += checkboxy('settings_show_flopps_link', 'Show Flopp\'s Map links in sidebar and under the "Additional Waypoints"') + show_help3("If there are no additional waypoints only the link in the sidebar is shown.") + "<br>";
+            html += newParameterOn3;
```

Comment:
Hier ist schon in der letzten Version etwas durcheinander gekommen. Bitte korrigiere noch die Zeilen wie folgt:
- Neue Zeile vor Zeile 12441
`            html += newParameterVersionSetzen(0.8) + newParameterOff;`
- Zeile 12441
`            html += newParameterOn1;`
- Zeile 12468
`            html += newParameterOn1;`

Reply:
Komma ist ergänzt

Reply:
Ich komm hier gerade nicht klar. Wenn ich die Änderungen so mache wie du sagst bzw. wie ich dein geschriebenes interpretiere, dann zerschießt es mir die Config. Ist es möglich, dass du die Änderung selbst hier im PR pushst?

Reply:
Ja, mach ich. Ich werde den PR mergen und dann meine Änderungen noch nachziehen.